Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -483,63 +483,63 @@ Enabled filters have priority from top to bottom in your list. Filter precedence
     <!-- Media general group -->
     <string name="settings_group_saving">Media saving</string>
     <string name="setting_save_image_board_folder">Save images in board folders</string>
-    <string name="setting_save_image_board_folder_description">Create a folder for each board when saving individual images</string>
+    <string name="setting_save_image_board_folder_description">One folder for each board to save individual images in.</string>
     <string name="setting_save_image_thread_folder">Save images in thread folders</string>
-    <string name="setting_save_image_thread_folder_description">Create a folder for each thread when saving individual images</string>
+    <string name="setting_save_image_thread_folder_description">One folder for each thread to save individual images in.</string>
     <string name="setting_save_album_board_folder">Save albums in board folders</string>
-    <string name="setting_save_album_board_folder_description">Create a folder for each board when saving image albums</string>
+    <string name="setting_save_album_board_folder_description">One folder for each board to save image albums in.</string>
     <string name="setting_save_album_thread_folder">Save albums in thread folders</string>
-    <string name="setting_save_album_thread_folder_description">Create a folder for each thread when saving image albums</string>
+    <string name="setting_save_album_thread_folder_description">One folder for each thread to save image albums in.</string>
     <string name="setting_save_server_filename">Save server filename</string>
-    <string name="setting_save_server_filename_description">Save the image with the filename the site assigned. If disabled, save the image with the filename the uploader assigned.</string>
+    <string name="setting_save_server_filename_description">Give saved filenames the name the site assigned. Otherwise the filename the uploader assigned is used.</string>
     <string name="setting_video_default_muted">Start videos muted</string>
     <string name="setting_video_stream">Stream videos</string>
-    <string name="setting_video_stream_description">Attempt to stream videos, instead of downloading them completely. (Works really bad with 4chan servers)</string>
-    <string name="setting_video_default_muted_description">If a video has audio, mute it by default</string>
+    <string name="setting_video_stream_description">Attempt to stream videos, instead of downloading them completely. (Works really badly on 4chan servers.)</string>
+    <string name="setting_video_default_muted_description">Videos are started with no audio.</string>
```

Comment:
"setting_video_default_muted">Start videos muted" already says "muted", which means saying it again doesn't explain anything. Instead of the redundant "muted", there is now a chance of understanding what "muted" is.